.github/workflows/ci-sharkfuser.yml

```diff
@@ -90,12 +91,22 @@ jobs:
       working-directory: ${{ env.SHARKFUSER_DIR }}
       run: |
         sudo apt update
-        sudo apt install cmake ninja-build catch2 ${{matrix.additional-packages}}
+        sudo apt install cmake ninja-build catch2 wget ${{matrix.additional-packages}}
         python -m venv --prompt fusili .venv
         source .venv/bin/activate
         python -m pip install --upgrade pip
         pip install -r ./test_requirements.txt
 
+    - name: Install prebuilt iree
+      working-directory: ${{ env.SHARKFUSER_DIR }}
+      run: |
+        wget --no-verbose -O iree-dist.tar.xz $IREE_RELEASE_URL
+        mkdir iree-dist
+        tar -xf iree-dist.tar.xz -C iree-dist
+        rm iree-dist.tar.xz
+        sudo mv iree-dist/* /usr/local/
```

Comment:
Will this work? `iree-dist` when unpacked has its own `bin`, `include`, and `lib` dirs.

Reply:
No this doesn't work it tries to overwrite the files which isn't correct.